frontend/src/CreateEvent/CreateEvent.module.css

```diff
@@ -0,0 +1,99 @@
+.container {
+  margin: 80px;
+  margin-right: 15vw;
+  display: flex;
+  float: right;
+}
+
+.back {
+  width: 50px;
+  height: 50px;
+  background-color: hsl(0, 0%, 85%);
+  border-radius: 25px;
+  margin: 16px;
+  margin-top: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 0px;
+}
+
+.back:hover {
```

Comment:
when changing background on hover, it's often a good idea to also add a transition in the original non-hover style so the background+colour changes smoothly.

```css
transition: background 0.2s linear, color 0.2s linear; 
```